nodewatcher/modules/devices/tplink/cpe.py

```diff
@@ -3,13 +3,13 @@
 from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
 
 
-class TPLinkCPE210(cgm_devices.DeviceBase):
+class TPLinkCPE210v1(cgm_devices.DeviceBase):
     """
-    TP-Link CPE210 device descriptor.
+    TP-Link CPE210 v1 device descriptor.
     """
 
-    identifier = 'tp-cpe210'
-    name = "CPE210"
+    identifier = 'tp-cpe210-v1'
```

Comment:
Existing identifier change requires a database migration as this will break all nodes using the old device identifier.

For an example of how to write a migration, see [this migration](https://github.com/wlanslovenija/nodewatcher/blob/development/nodewatcher/modules/devices/migrations/0001_change_glinet_id.py).

Reply:
Ok, will make necessary DB migrations today

Reply:
Ok,I made the migrations manually since Django wont detect them.
Only thing that I dont know to configure is dependencies